### ISSUE TITLE
Peppol integration (B2G and B2B e-Invoicing standard)

### ISSUE THREAD

**Issue by YordanSoares** [wpovernight/woocommerce-pdf-invoices-packing-slips#721]
[A user asked](https://wordpress.org/support/topic/peppol-intehgration/) if we have integration with Peppol, an e-Invoicing standard that is used for B2G and B2B.

Please note that Belgian government planned to make it required to B2B for 2026.

References:
- https://peppol.org/
- https://edicomgroup.com/blog/belgium-will-make-b2b-electronic-invoice-mandatory

**Comment by nieelu** [comment]
Hi, is there any update regarding this?
We are a Belgian company and need to integrate with Peppol as of 2026.

**Comment by alexmigf** [comment]
Hi @nieelu 

Yes, we're still in the early stages, but progress is underway.

**Comment by alexmigf** [comment]
I'm wrapping up the refactor of our [e-invoicing code](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/1149)—Peppol is next on my list.

**Comment by alexmigf** [comment]
Yes, we're currently finalizing the implementation details and expect to start work very soon.

**Comment by alexmigf** [comment]
Roadmap update: we’re targeting the end of this month to begin testing.

**Comment by alexmigf** [comment]
The development phase is complete, and we are now moving on to internal testing before rolling it out to a few customers. We are getting close and appreciate your patience.

**Comment by alexmigf** [comment]
We’ll begin testing with a few customers in the coming days. This feature won’t be included in the free plugin for now, so anyone interested should [reach out to us directly](https://wpovernight.com/contact/).

**Comment by alexmigf** [comment]
Unfortunately, we’re facing a few final challenges with customer identity validation that are slowing things down. We’re fully aware of the timeline and are doing our utmost to have everything ready in the next few days. We sincerely apologize for the delay.

**Comment by xian-bits** [comment]
Hello i sended an email with the form u linked. Can i get access asap? Almost end of year and we the company dont have much time left. And the client really wants to see it work. When can i expect the invitation?

**Comment by alexmigf** [comment]
Hi @xian-bits 

We are sending invitations in stages while we make small adjustments based on feedback from current customer testing. You should receive your invitation in the coming days.

**Comment by creativitijd** [comment]
Hello,

How long does it take before my clients get approved.
It has been over 24 hours for all 6 clients.


Met vriendelijke groeten,



Maarten Geuns



Zaakvoerder Creativitijd



***@***.******@***.***>

0498 82 21 39

www.creativitijd.be<https://www.creativitijd.be>



Noordervest 13

3990 Peer

België



BTW BE 0678 58 25 04



[signature_759453879]



Van: Alexandre Faustino ***@***.***>
Datum: woensdag, 24 december 2025 om 12:49
Aan: wpovernight/woocommerce-pdf-invoices-packing-slips ***@***.***>
CC: Maarten Geuns ***@***.***>, Mention ***@***.***>
Onderwerp: Re: [wpovernight/woocommerce-pdf-invoices-packing-slips] Peppol integration (B2G and B2B e-Invoicing standard) (Issue #721)

[https://avatars.githubusercontent.com/u/533273?s=20&v=4]alexmigf left a comment (wpovernight/woocommerce-pdf-invoices-packing-slips#721)<https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/issues/721#issuecomment-3689582028>

We’ve decided to open the Peppol subscription for purchase. You can buy it here: https://wpovernight.com/downloads/woocommerce-edocuments-peppol/

We’re currently preparing the documentation. In the meantime, feel free to contact us<https://wpovernight.com/contact/> if you have any questions.

**Comment by xian-bits** [comment]
Any idea why its pending since 22/12?

Status | Pending
2025-12-22 13:52
